recipes/django-material/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "django-material" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "5ad516f55f05e2fabb31797d4c5ae5819ae37c8945386d947f09eb550bf4463b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+
+about:
+  home: http://forms.viewflow.io/
+  license: BSD 3-Clause
+  license_file: LICENSE.txt
```

Comment:
Ah.  `LICENSE `is listed in the manifest.in, but is not included in the pypi build, presumably because pypi was looking for `LICENSE `instead of the `LICENSE.txt` found on the github repo. Does that mean it'll need to be re-built for pypi?